### DISCUSSION TITLE
M3 Plus prusaslicer profile

### DISCUSSION THREAD

**Discussion by cubiq** [sn4k3/UVtools#470] [Q&A] [answered]
Just curious about the width/height set by the UVTools profile for the Anycubic M3+. They are respectively 198.15 and 123.84mm but the size on the official Anycubic workshop is 197 and 122.8mm.

Is it an error on UVTools side or is there a reason for the discrepancy?

**Comment by sn4k3** [comment, marked as answer]
It's all about math and keep correct pixel pitch. M3 Plus pixel size: **34.400**um

UVtools:
```
198.15 / 5760 * 1000 = 34,40 um
123.84 / 3600 * 1000 = 34,40 um
```

Anycubic:
```
197.0 / 5760 * 1000 = 34.20 um
122.8 / 3600 * 1000 = 34,11 um
```

That doesn't mean anycubic had it wrong, they can offset internally and make it right but who knows...
Using same anycubic settings on PS will cause minor distortion on the printed model, because pixel size is not correct.
On some printers they have the correct values to give the proper pixel size, while others don't, it's a mess...

**Comment by cubiq** [comment]
so I made some tests, it seems that anycubic workshop (and lychee) sets the PixelSizeUm to 34.4 in the header even if the W/H are 197.0 x 122.8. I'm guessing the pixelratio is automatically calculated to 34.201 if not present in the header ?

I don't know if it makes any difference, but one could set the pixel ratio with a parameter in the notes. Anyway right now you have to keep the values suggested by the UVtools.

**Reply by sn4k3** [reply]
That field is just informative and dont matter for the printer, it is set static by slicers, they don't calculate that based on your settings. You can test by input other values on  anycubic or lychee and you will note that pixelsizeum will probably keep 34.4um even if not.

At slice time all it matters is the resolution and display size
You can see more about this here: https://github.com/sn4k3/UVtools/issues/369
